tests/tests/test_timetable.py

```diff
@@ -14,14 +14,27 @@ class _TrainScheduleDetails:
     id: int
     train_name: str
     departure_time: int
-    train_path: int
+    arrival_time: float
+    path_id: int
+    timetable_id: int
+    rolling_stock_id: int
+    initial_speed: int
+    labels: Iterable[str]
+    allowances: Iterable[dict]
+    speed_limit_tags: str
+    comfort: str
+    options: dict
+    power_restriction_ranges: Iterable[dict]
+    mechanical_energy_consumed: Iterable[dict]
+    stops: int
+    path_length: float
 
 
 @dataclass(frozen=True)
 class _TimetableResponse:
     id: int
     name: str
-    train_schedules: Iterable[_TrainScheduleDetails]
+    detailed_train_schedules: Iterable[_TrainScheduleDetails]
```

Comment:
At this point it's still an iterable of dicts, it's converted to `_TrainScheduleDetails` later on